leaderelection/leader_election.go

```diff
@@ -39,6 +40,9 @@ const (
 	defaultLeaseDuration = 15 * time.Second
 	defaultRenewDeadline = 10 * time.Second
 	defaultRetryPeriod   = 5 * time.Second
+	healthCheckTimeout   = 20 * time.Second
+
+	HealthCheckerAddress = "/healthz/leader-election"
```

Comment:
Is there a doc anywhere that shows where this endpoint is used? Is it configurable?

Reply:
It's not configurable and I'm leaning towards exposing less knobs until there's a need. The documentation of `RegisterHealthCheck` mentions this constant, although I could explicitly state this URL if you think that could be better. It's up to the caller to document the URL in flag descriptions.

Reply:
Yeah if you could add a comment just pointing to where it's documented, that would be helpful for someone looking at this.

Reply:
Done (if I interpreted your suggestion correctly)

Reply:
Sorry I meant pointing to where this is defined in the leader election library.